fractions/netflix/pom.xml

```diff
@@ -26,7 +26,7 @@
     <version.netflix.commons>0.1.1</version.netflix.commons>
     <version.archaius>0.6.6</version.archaius>
     <version.ribbon>2.1.0</version.ribbon>
-    <version.hystrix>1.4.21</version.hystrix>
+    <version.hystrix>1.5.12</version.hystrix>
```

Comment:
This is not enough IMHO. `hystrix-code` depends on `HdrHistogram`.

Reply:
Can you elaborate on this?